l2geth/rollup/dump/constants.go

```diff
@@ -9,4 +9,4 @@ var BvmFeeWallet = common.HexToAddress("0x42000000000000000000000000000000000000
 var TssRewardAddress = common.HexToAddress("0x4200000000000000000000000000000000000020")
 var DeadAddress = common.HexToAddress("0xdeaDDeADDEaDdeaDdEAddEADDEAdDeadDEADDEaD")
 var BvmWhitelistAddress = common.HexToAddress("0x4200000000000000000000000000000000000002")
-var BvmReorgAddress = common.HexToAddress("0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD2222")
+var BvmRollbackAddress = common.HexToAddress("0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD2222")
```

Comment:
is this necessary to have this type of address? i think rollback logic will have no transaction to execute, it only triggered by some event context